rmc/utils/missense_badness.py

```diff
@@ -140,6 +42,7 @@ def prepare_amino_acid_ht(
         - Add observed and OE annotation
         - Write to `amino_acids_oe` resource path
 
+    :param freeze: RMC freeze number. Default is CURRENT_FREEZE.
```

Comment:
this is ordered first in the docstring but last in the args -- either order works for me, but this should probably be consistent

Reply:
Yep, good catch!